pkg/driver/driver.go

```diff
@@ -49,6 +55,16 @@ func New(config Config) (*Driver, error) {
 	case config.WorkloadAPISocketDir == "":
 		return nil, errors.New("workload API socket directory is required")
 	}
+
+	// Set the SELinux label on the workload API directory. This allows the
+	// mount to be used within OpenShift, for example. This will fail if the
+	// Workload API socket directory is mounted read-only.
+	if err := chcon(config.WorkloadAPISocketDir, seLinuxContainerFileLabel, true); err != nil {
```

Comment:
Would it make sense to check if SELinux is in enforcing mode before attempting this? That would allow users without enforcing SELinux to use a read-only mount and avoid log errors.

Reply:
That seems reasonable. I'll look into how to do that.